### ISSUE TITLE
Weird output with FormHelper

### ISSUE THREAD

**Issue by grandmanitou** [Holt59/cakephp3-bootstrap-helpers#82]
My Code:
`<?= $this->Form->submit(__('Login'), ['class' => 'btn btn-primary']); ?>`

Expected:
`<div class="form-group"><input type="submit" class="btn btn-primary" value="Login"></div>
`

Output:
`<div class="form-group"><input type="submit" class="btn btn-primary btn btn-default" value="Login"></div>`

Either it's a bug or I am missing somthing.

Thanks

**Comment by Holt59** [comment]
The `btn` class is added automatically to any button created using the `button()` and `submit()` method, and the `btn-xxxx` class is added using the `bootstrap-type` option:

```php
 <?= $this->Form->submit(__('Login'), ['bootstrap-type' => 'primary']); ?>
```

If you don't specify the `bootstrap-type` option, the `btn-default` class will be chosen. You can update the default type using `$this->Form->config('buttons.type', 'primary');`.

**Comment by grandmanitou** [comment]
Thanks

**Comment by Holt59** [comment]
I am reopening this because this is not very intuitive to have the helpers override the `btn-primary` class if the `bootstrap-type` option is not here.

**Comment by Holt59** [comment]
This is fixed by 514266d8845f3338c4dbceb4ff20c74b076115fe, I would encourage you not to add the `btn` class because it's redundant but I understand that the `bootstrap-type` options is not that convenient... If  you have a global style on your site, the best option is to use the `buttons.type` option (you can specify it when loading the helper as with any option)

I am sorry the option part is not yet documented but this is a new feature and I do not had time to update the documentation yet...
